src/utils/onchain-data.ts

```diff
@@ -60,90 +57,13 @@ export async function getSarcophagiIds(web3Interface: Web3Interface): Promise<st
   );
 }
 
-export async function getOnchainCursedSarcophagi(
-  web3Interface: Web3Interface
-): Promise<SarcophagusData[]> {
-  const archSarco: SarcophagusData[] = [];
-
-  inMemoryStore.gracePeriod = inMemoryStore.gracePeriod || await getGracePeriod(web3Interface);
-
-  const sarcoIds = await web3Interface.viewStateFacet.getArchaeologistSarcophagi(
-    web3Interface.ethWallet.address
-  );
-
-  sarcoIds.map(async sarcoId => {
-    const sarco = await web3Interface.viewStateFacet.getSarcophagus(sarcoId);
-    const archStorage = await web3Interface.viewStateFacet.getSarcophagusArchaeologist(
-      sarcoId,
-      web3Interface.ethWallet.address
-    );
-
-    // If sarcophagus exists and the unencrypted shard is empty, then it (potentially)
-    // still needs to be unwrapped
-    if (sarco.state === SarcophagusState.Exists && archStorage.unencryptedShard === '0x') {
-      const nowTimestampInSeconds = new Date().getTime() / 1000;
-
-      const tooLateToUnwrap =
-        (sarco.resurrectionTime.toNumber() + inMemoryStore.gracePeriod!.toNumber()) <
-        nowTimestampInSeconds;
-
-      if (tooLateToUnwrap) {
-        archLogger.warn(`You failed to unwrap a Sarcophagus on time: ${sarcoId}\n`);
-        // TODO: archaeologist might want to call clean here
-
-        return;
-      }
-
-      // only add unwrappable sarco that have state === EXISTS and haven't already unwrapped (uploaded their unencrypted shard)
-      const resurrectionTime = new Date(sarco.resurrectionTime.toNumber() * 1000);
-      archSarco.push({
-        id: sarcoId,
-        resurrectionTime,
-      });
-
-      // Schedule an uwrap job for each sarco this arch is cursed on. `scheduleUnwrap` will cancel existing
-      // schedules, so no duplicate jobs will be created.
-
-      // NOTE: It is marginally possible that the arch node and a sarco it's bonded to end up in a state where
-      // the sarco's resurrection time is past, but we're still within its grace period. In that case,
-      // scheduling a job won't work as resurrection time is in the past, so we'll want immediately attempt
-      // a rewrap in an effort to salvage the situation.
```

Comment:
I feel like these comments are important to leave in. They've been removed in `refresh-data.ts:43`

Reply:
re-added!